src/utils/page.ts

```
@@ -53,3 +53,15 @@ export function getReferrer() {
     },
   };
 }
+
+export function generateTriggerCTAURL(campaignID: string, originalURL: string): string {
+  if (!campaignID) throw new Error('campaignID must not be empty');
```

Im not sure we want to throw a global error (that will likey kill difi on the page), can we not just do `return originalURL` ?

---

a17048f